packages/react-select/src/__tests__/Select.test.js

```diff
@@ -812,6 +812,26 @@ cases(
       selectedOption: OPTIONS_DISABLED[0],
       nextFocusOption: OPTIONS_DISABLED[1],
     },
+    'single select > disabled options should be skipped, ArrowDown': {
+      props: {
+        menuIsOpen: true,
+        options: OPTIONS_DISABLED,
+        skipDisabled: true
+      },
+      keyEvent: [{ keyCode: 40, key: 'ArrowDown' }],
+      selectedOption: OPTIONS_DISABLED[0],
+      nextFocusOption: OPTIONS_DISABLED[2],
+    },
+    'single select > disabled options should be skipped, ArrowUp': {
```

Comment:
Hey @luke-webdev! Thanks for this PR, it looks good, however the second test is failing:

```
  ● focus on options > keyboard interaction with Menu › single select > disabled options should be skipped, ArrowUp

    expect(received).toEqual(expected) // deep equality

    Expected: "2"
    Received: "3"

      847 |     expect(
      848 |       container.querySelector('.react-select__option--is-focused').textContent
    > 849 |     ).toEqual(selectedOption.label);
          |       ^
      850 |
      851 |     for (let event of keyEvent) {
      852 |       fireEvent.keyDown(container.querySelector('.react-select__menu'), event);

      at toEqual (packages/react-select/src/__tests__/Select.test.js:849:7)
      at Object.cb (node_modules/jest-in-case/index.js:40:20)
```

For some reason it thinks that the focused item is the 3rd option, not the 2nd. I've attempted to debug this but I can't see anything obvious. Any ideas?

Reply:
Hi @bladey, thanks for taking the time to review the code. I have fixed the test that was failing in this branch.

Let me know if there are any other issues with the pull request. Thanks!